app/scripts/views/mixins/password-mixin.js

```diff
@@ -124,6 +130,16 @@ define(function (require, exports, module) {
       }
     },
 
+    setVisibilityShowPassword (el) {
+      const $passwordField = $(el);
+      const $showPasswordLabel = $passwordField.siblings('.show-password-label');
```

Comment:
The name of this function and what it does do not fully align, and the input value does not at all follow from the name.

This function is really `updatePasswordsShowVisibility` or similar.